worlds/ladx/__init__.py

```diff
@@ -136,7 +136,7 @@ def convert_ap_options_to_ladxr_logic(self):
         world_setup = LADXRWorldSetup()
         world_setup.randomize(self.ladxr_settings, self.random)
         self.ladxr_logic = LADXRLogic(configuration_options=self.ladxr_settings, world_setup=world_setup)
-        self.ladxr_itempool = LADXRItemPool(self.ladxr_logic, self.ladxr_settings, self.random).toDict()
+        self.ladxr_itempool = LADXRItemPool(self.ladxr_logic, self.ladxr_settings, self.random, self.options.stabilize_item_pool).toDict()

```

Comment:
Pretty sure you want the bool here and not the option instance itself
```suggestion
        self.ladxr_itempool = LADXRItemPool(self.ladxr_logic, self.ladxr_settings, self.random, bool(self.options.stabilize_item_pool)).toDict()

```